src/lib/helpers.js

```diff
@@ -23,3 +23,15 @@ export const getRandomKeyString = () =>
   Math.random()
     .toString(36)
     .substring(2, 15)
+
+export const getCompleteFolderPath = (defaultDir, konnectorName, t) => {
+  let folderPath = `/${defaultDir}/$konnector`
+  return folderPath
+    .replace(/\/\//g, '/')
+    .replace(/\$konnector/gi, konnectorName)
```

Comment:
In cozy-harvest, we will need to keep the konnectorName translated.

Reply:
We don't have much translated connectors for the now we can still the name from manifest. But completely agree about doing that in cozy-harvest 👍